### PR TITLE
CEDS-3577 Add error message for Y999 doc code

### PR DESCRIPTION
**Pull request by magicaljellybeans**
Adding of temporary error message stopping use of code "Y999"

Had to enable escaping of HTML in a couple of places to allow for this multi-line error message.

### CODE REVIEW

File: app/views/components/gds/errorSummary.scala.html [hunk 1]
```diff
@@ -22,6 +22,12 @@
 
 @errorList = @{errors.map { error =>
     val errorKey = error.key.replace(".", "_").replace('[', '_').replace("]", "")
+    if (error.message.contains("Y999"))
```
Comment (udinobi) [inline app/views/components/gds/errorSummary.scala.html:25]: What about...
```
    ErrorLink(
        href = Some(s"#${errorFieldName.getOrElse(errorKey)}"),
        content =
            if (error.message.contains("Y999")) HtmlContent(messages(error.message, error.args: _*))
            else Text(messages(error.message, error.args: _*))
    )
```

**Review by udinobi — DISMISSED** [review]

**Review by glidester — CHANGES_REQUESTED** [review]
Can we think about generalising this 'errors as HTML' feature so it is not 'Y999' specific (probably need to use this for other errors in future). :slightly_smiling_face:

**Comment by magicaljellybeans** [comment]
Thanks for the pointers, changes made.

**Review by glidester — APPROVED** [review]
:+1:

**Review by udinobi — APPROVED** [review]

**Pull request merged**